src/lib/modules/Relations/backoffice/components/RelationRemover/RelationRemover.js

```diff
@@ -39,7 +39,12 @@ export default class RelationRemover extends Component {
       >
         <Modal.Header>Confirm removal</Modal.Header>
         <Modal.Content>
-          Are you sure you want to delete this relation?
+          <p>Are you sure you want to delete this relation?</p>
+          <p>
+            Please note that by deleting this {relationType} relation, you
```

Comment:
I still find this sentence quite confusing. Is it only me? ping also @kprzerwa @zzacharo @NRodriguezcuellar

For the title, I would use `remove` and not `delete`:
```
Are you sure you want to remove this relation?
```
For the body:
```
Please note that all {relationType} relations to this literature will be removed.
```
I would not add the title because it does not contain the relation type, e.g. language, and it is a bit misleading.
e.g. `Please note that all language relations to "The Lord Of The Rings" will be removed.`
while it should be `Please note that all language relations to "The Lord Of The Rings - EN" will be removed.`

Reply:
As far as I understand, we are already using the "Please note that all language relations to "The Lord Of The Rings - EN" will be removed." format. In the current implementation, the message already alludes to the specific edition or language version of the document whose relations will be removed.

Reply:
Implemented, as discussed.
![Screenshot from 2021-09-15 15-24-11](https://user-images.githubusercontent.com/25476209/133441529-7962cbbc-7091-4e60-a8ad-fa4ac4b02538.png)
![Screenshot from 2021-09-15 15-24-22](https://user-images.githubusercontent.com/25476209/133441533-99cdb95e-fffc-4cba-844f-ebcf501e8469.png)